flux/clusters/k8s-01/keycloak-db/base/postgrescluster.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: db-keycloak
+  labels:
+    app.kubernetes.io/component: database
+    app.kubernetes.io/managed-by: flux
+    app.kubernetes.io/name: db-keycloak
+    app.kubernetes.io/part-of: keycloak
+spec:
+  databaseInitSQL:
+    key: init.sql
+    name: keycloak-db-init-sql
+  backups:
+    pgbackrest:
+      image: 'registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.40-1'
+      repos:
+        - name: repo1
+          volume:
+            volumeClaimSpec:
+              accessModes:
+                - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 50Gi
+              storageClassName: local-path-regular
+  instances:
+    - affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchLabels:
+                    postgres-operator.crunchydata.com/cluster: db-keycloak
+                    postgres-operator.crunchydata.com/instance-set: '01'
+      dataVolumeClaimSpec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 50Gi
+      name: '01'
+      replicas: 1
+      resources:
```

Comment:
Is it possible to check KK RDS instance type to see mem/cpu used?

Reply:
This is Amazon Aurora Serverless services, so we can't check it. We can only check the configuration of the DB cluster: Minimum Aurora capacity units: 1 ACU (2GiB RAM), Maximum Aurora capacity units: 16 (32 GiB RAM). Current capacity: 1 ACU. 
https://aws.amazon.com/rds/aurora/serverless/